cmd/notation/verify.go

```diff
@@ -26,6 +26,13 @@ type verifyOpts struct {
 	reference    string
 	pluginConfig []string
 	userMetadata []string
+	outputFormat string
+}
+
+type verifyOutput struct {
+	Reference    string            `json:"reference"`
+	UserMetadata map[string]string `json:"userMetadata"`
```

Comment:
should we add omitempty to not not print userMetadata if there is no userMetadata?

Reply:
good call, will update